cmake/USDPluginTools.cmake

```diff
@@ -491,7 +446,7 @@ function(_usd_cpp_library NAME)
             add_custom_command(
                 TARGET ${NAME}
                 POST_BUILD
-                    COMMAND ${CMAKE_COMMAND} -E create_symlink $<TARGET_FILE:${NAME}> ${PROJECT_BINARY_DIR}/${LIBRARY_INSTALL_PREFIX}/$<TARGET_LINKER_FILE_NAME:${NAME}>
+                    COMMAND ${CMAKE_COMMAND} -E create_symlink $<TARGET_FILE:${NAME}> ${PROJECT_BINARY_DIR}/${LIBRARY_INSTALL_PREFIX}/$<TARGET_FILE_NAME:${NAME}>
```

Comment:
I think with this change we can remove the `if (args_TYPE STREQUAL "PLUGIN")` condition as both add_custom_command(s) are setting the exact same POST_BUILD COMMAND property!